release_notes/release_notes.html

```diff
@@ -1,5 +1,11 @@
-<b>Panorama Release Notes - Published by Splunk April 04, 2022</b>
+<b>Panorama Release Notes - Published by Splunk May 02, 2022</b>
```

Comment:
It looks like these dates are a bit off, but I think it's no longer important since we're not publishing to the phantom portal.

Reply:
As per our discussion on Slack thread, I'm resolving this thread, as the date on the readme.html is no longer being used on Splunkbase.